src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -317,12 +328,21 @@ void InferRequest::SetBlob(const std::string& name, const Blob::Ptr& data) {
                 if (compoundBlobPassed) {
                     IE_THROW(NotImplemented) << cannot_set_compound;
                 }
-                size_t blobSize = desc.getLayout() != SCALAR
-                    ? details::product(desc.getDims())
-                    : 1;
-                if (dataSize != blobSize) {
-                    IE_THROW() << "Input blob size is not equal network input size ("
-                               << dataSize << "!=" << blobSize << ").";
+                if (foundInput->getInputData()->isDynamic()) {
```

Comment:
This method is deprecated

Reply:
Which method should I use instead to find out that this input contains dynamic dims?

Reply:
Fixed